docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -127,9 +127,31 @@ When the support to critical pods is enabled and the pods in a DaemonSet are
 labelled as critical, the Daemon pods are created with an additional
 `NoSchedule` toleration for the `node.kubernetes.io/out-of-disk` taint.
 
-Note that all above `NoSchedule` taints above are created only in version 1.8 or later if the alpha feature `TaintNodesByCondition` is enabled.
+Note that all above `NoSchedule` taints above are created only in version 1.8 or later
+ if the alpha feature `TaintNodesByCondition` is enabled.
 
-Also note that the `node-role.kubernetes.io/master` `NoSchedule` toleration specified in the above example is needed on 1.6 or later to schedule on *master* nodes as this is not a default toleration.
+Also note that the `node-role.kubernetes.io/master` `NoSchedule` toleration specified in the above example
+ is needed on 1.6 or later to schedule on *master* nodes as this is not a default toleration.
+
+In version 1.10, an alpha feature, named `ScheduleDaemonSetPods`, was introduced to schedule DaemonSet Pods
+by default scheduler instread of DameonSet controller.
+
+ When the feature was enabled, the following `NodeAffinity` term will be added to the DaemonSet Pods;
+ so the default scheduler will bind the Pod to the target host by `NodeAffinity` feature.
```

Comment:
Edit:
When this feature is enabled, the `NodeAffinity` term is added to the DaemonSet Pods. This enables the default scheduler to bind the Pod to the target host.

Reply:
It's clear from the example that follows, but if you think it's needed, then something like this is clearer:
When this feature is enabled, the NodeAffinity term is added to the DaemonSet Pods, as in the following example. This enables the default scheduler to bind the Pod to the target host.